issue#254_add_PGID-UID_to_Rockon_doc (#255)
rockstor/rockstor-doc: docker-based-rock-ons/overview.rst

to refresh the list of available Rock-ons. To install a listed Rock-on use
its **Install** button on the Rock-ons WebUI page.

> .. _user_group:

Reviewer: I would prefer further specification to differentiate it from our Users/Groups dedicated page:
```suggestion
.. _rockons_user_group:
```

UID and GID usage in Rock-ons
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> During the configuration of Rock-ons, quite a few require the specification of a User ID (UID) and/or a User Group ID (GID). Aside from the various options using the command line to determine existing or creating new UIDs or GIDs, the simplest way is to take advantage of the User and/or Group management screen. Navigate to System --> Users (or Groups if necessary).

Reviewer: Minor style edits (rst style recommends line length of 79 characters max):
```suggestion
During the configuration of Rock-ons, quite a few require the specification of a
User ID (UID) and/or a Group ID (GID). Aside from the various options using the
command line to determine existing UIDs and GIDs, the simplest way is to take
advantage of the User and/or Group management screens. Navigate to System
--> Users (or Groups if necessary).
```

.. image:: users_page.png
   :scale: 30%
   :align: center
   

> The two highlighted columns show the UID and GID. For example, the user plex had the UID 1001, as well as the GID 1001, which could then be used to set up the Plex Rock-on.

Reviewer: Minor style edits:
```suggestion
The two highlighted columns show the UID and GID. In the example above, the
user :code:`plex` has the UID 1001, as well as the GID 1001, which could then
be used to set up the Plex rock-on.
```
